docusaurus/docs/develop/developer_guide/adding_params.md

```diff
@@ -43,245 +50,468 @@ https://github.com/ignite/cli/issues/3684#issuecomment-2299796210
 
 ## Step-by-Step Instructions
 
+:::warning
+The steps outlined below follow **the same example** where:
+- **Module name**: `examplemod`
+- **New parameter name**: `new_param`
+- **Default value**: `42`
+
+When following these steps, be sure to substitute these example values with your own!
+:::
+
+### 0. If the Module Doesn't Already Support a `MsgUpdateParam` Message
+
+In order to support **individual parameter updates**, the module MUST have a `MsgUpdateParam` message.
+If the module doesn't already support this message, it will need to be added.
```

Comment:
```suggestion
If the module doesn't already support this message, it will need to be added.

:::tip
At any point, you can always run `go test ./x/examplemod/...` to check whether everything is working or locate outstanding necessary changes.
:::
```